docs/configuration_and_usage.rst

```diff
@@ -0,0 +1,106 @@
+Configuration and usage
+=======================
+
+Getting around the administrator interface
+------------------------------------------
+
+``Edx-enterprise`` is integrated with edx-platform Django admin. It can be accessed at
+``$LMS_SERVER_ADDRESS/admin/enterprise``. Current administrator interface is considered provisional until an independent
+full-featured enterprise portal is built, so not much time was spent on improving admin tools UX.
+
+.. image:: images/enterprise-admin.png
+
+So far, there are four items in the admin interface:
+
+* "Enrollment notification email templates" - manages templates used to build enrollment notifications.
+* "Enterprise Customers" - lists and manages Enterprise Customer records.
+* "Enterprise Customer Users" - lists and manages enterprise customer students.
+* "Pending Enterprise Customer Users" - lists and manages "pending" enterprise customer students.
+
+.. image:: images/enterprise-customer.png
+
+The edx-enterpise admin site provides two ways to manage learners: The "Enterprise Customer Users" section mentioned
+above, and the "Manage Learners" view. The "Manage Learners" view is recommended as it is easier to understand and
+provides more features. To access it, first go to the "Enterprise Customer" section of the admin site, then click on a
+customer, and then click on the "Manage Learners" button in the top-right corner of the page.
+
+Manage Learners View
+--------------------
+
+The Manage Learners view is meant to be used by non-technical staff to administer enterprise learners.
+It can be used to:
+
+* Associate ("link") and disassociate ("unlink") existing students with Enterprise Customer.
+* Associate learners with the Enterprise customer even if they don't have an account yet (these
+  learners are known as "Pending Enterprise Customer Users"). Pending Enterprise Customer Users are
+  referenced/identified by their email address only.
+* Manually enroll groups of enterprise learners into courses and/or programs.
+
+.. image:: images/manage-learners.png
+
+Integrating with Course Catalog
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order for course enrollment features to work, ``edx-enterprise`` talks to the `Course Catalog Service`_. Instructions
+on setting up and configuring Course Catalog can be found at corresponding ``docs section`` of Course Catalog and in
```

Comment:
The RST formatting of "docs section" is not correct - it's not showing up as a link in the GitHub preview. (But "Course Catalog Service" is.)